test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -48,6 +48,20 @@ Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @(
     }
 }
 
+Describe "Validate that the Help function can Run in strict mode" -Tags @('CI') {
+
+    It "Help doesn't fail when strict mode is on." {
+
+        $help = & {
+            # run in nested scope to keep strict mode fromo affecting other tests.
+            Set-StrictMode -Version Latest 
+            Help
+        }
+        # the help function renders the help content as text so just verify that there is content
+        [bool] $help | Should -Be $true 
```

Comment:
Perhaps: `$help | Should -Not -BeEmptyOrNull` instead?  If it should fail, the test failure output may make it easier to understand what is going on